doc/api/packages.md

```diff
@@ -547,6 +542,49 @@ exports, while resolving the existing `"node"`, `"default"`, `"import"`, and
 
 Any number of custom conditions can be set with repeat flags.
 
+### Conditions Definitions
+
+The `"import"`, `"require"`, `"node"` and `"default"` conditions are defined
+and implemented in Node.js core,
+[as specified above](#esm_conditional_exports).
+
+Other condition strings are unknown to Node.js and thus ignored by default.
+Runtimes or tools other than Node.js can use them at their discretion.
+
+The following condition definitions are currently endorsed by Node.js:
+
+* `"browser"` - any environment which implements a standard subset of
+   global browser APIs available from JS in web browsers, including the DOM
+   APIs.
+* `"development"` - can be used to define a development-only environment
+   entry point. _Must always be mutually exclusive with `"production"`._
+* `"production"` - can be used to define a production environment entry
+   point. _Must always be mutually exclusive with `"development"`._
```

Comment:
It seems a little weird that we’re saying that these two conditions must be mutually exclusive, while Node.js completely ignores them. Are what we really saying is that Node.js recommends that people use these two strings and if they’re used, that they should be mutually exclusive? But really it’s all just a recommendation?

Reply:
Yes exactly, we are defining the conditions and how they should be used.

Reply:
That’s fine. I was just confused when I read it, like why would it be so specific if it’s just a recommendation. I’m not sure what the solution is, if it needs to change at all, but I thought I’d mention that it confused me.